.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: ruby
+
+rvm:
+  - 2.1
+  - 2.2
```

Comment:
Could you add 2.3 and ruby-head?
And add ruby-head to allow_failures is better.